dark/assets/css/styles.css

```
@@ -1097,6 +1097,8 @@ html {
   flex-direction: column;
   align-items: center;
   justify-content: center;
+  background-color: rgba(0, 0, 0, 0.8);

```

Don't reduce the bg color @Rishav1707 
```suggestion
  background-color: rgba(0, 0, 0, 0.9);
```